concepts/05 Widgets/DropDownBox/15 Synchronize with the Embedded Element.md

```diff
@@ -138,6 +121,63 @@ Assign the field's name to the **DropDownBox**'s [valueExpr](/api-reference/10%2
         }
         </script>
 
+    ##### React
+
+        import React from 'react';
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import { DropDownBox } from 'devextreme-react/drop-down-box';
+        import { DataGrid, Selection } from "devextreme-react/data-grid";
+        import ArrayStore from "devextreme/data/array_store";
+        
+        class App extends React.Component {
+            constructor(props) {
+                super(props);
+
+                dropDownBoxData = dropDownBoxData;
+                gridDataSource = new ArrayStore({
```

Comment:
Aren't `dropDownBoxData` and `gridDataSource` supposed to be constants declared outside the class?